Add CrossSettlement and ExternalIdentifier with new time filter on getAllAccountingItems (#265)
MewsSystems/gitbook-connector-api: operations/finance.md

| `ClientToken` | string | required | Token identifying the client application. |
| `AccessToken` | string | required | Access token of the client application. |
| `Client` | string | required | Name and version of the client application. |

> | `ConsumedUtc` | [Time interval](enterprises.md#time-interval) | optional, max length 1 months | Interval in which the [Outlet item](#outlet-item) was consumed. Required if no other filter is provided. |
> | `ClosedUtc` | [Time interval](enterprises.md#time-interval) | optional, max length 1 months | Interval in which the [Outlet bill](#outlet-bill) was closed. Required if no other filter is provided. |

Reviewer: Outlet bills and items are something different. Will explain in person.
```suggestion
| `ConsumedUtc` | [Time interval](enterprises.md#time-interval) | optional, max length 1 months | Interval in which the accounting item was consumed. Required if no other filter is provided. |
| `ClosedUtc` | [Time interval](enterprises.md#time-interval) | optional, max length 1 months | Interval in which the accounting item was closed. Required if no other filter is provided. |
```

> | `ItemIds` | array of string | optional, max 1000 items | Unique identifiers of the Accounting items. Required if `StartUtc` and `EndUtc` are not specified. |

Reviewer: ```suggestion
| `ItemIds` | array of string | optional, max 1000 items | Unique identifiers of the Accounting items. Required if `ConsumedUtc` or `ClosedUtc` are not specified. |
```

| `BillId` | string | optional | Unique identifier of an open bill of the customer where to assign the payment. |
| `Amount` | [Amount value](finance.md#amount-value) | required | Amount of the external card payment. |
| `Type` | string [External payment type](finance.md#external-payment-type) | optional | Type of the external payment. |

> | `ExternalIdentifier` | string | optional | Identifier of the external payment accepted as a pre-payment. |

Reviewer: This is way too specific description, this can be used in other way than just the one case from the issue.
```suggestion
| `ExternalIdentifier` | string | optional | Identifier of the payment from external system. |
```
